### PR TITLE
Fix script not working in docker container, added debug option, lint …

### PR DESCRIPTION
**Pull request by ojsef39**
Hey, i noticed that the script won't work in the docker container because the JSON is different.

What I've changed:
- Linted python file
- added isdocker function to check which parsing method has to be used
- added rolling-dev for dev branch to workflow
- added DEBUG env to see output variables
- added prometheus and grafana dashboard example

### CODE REVIEW

File: intel-gpu-exporter.py [hunk 1]
```diff
@@ -68,28 +117,54 @@ def update(data):
 
     igpu_rc6.set(data.get("rc6", {}).get("value", 0))
 
-if __name__ == '__main__':
-    logging.basicConfig(format='%(asctime)s - %(message)s', level=logging.INFO)
-    start_http_server(8080)
 
-    period = os.getenv("REFRESH_PERIOD_MS", 10000)
+def is_docker():
+    return os.path.exists("/.dockerenv")
```
Comment (onedr0p) [inline intel-gpu-exporter.py]: This should just be an env defined in the Dockerfile which you can check for in the script. No need to add or check for files.

**Review by onedr0p — CHANGES_REQUESTED** [review]

File: intel-gpu-exporter.py (hunk 1 above)
Comment (ojsef39) [reply intel-gpu-exporter.py]: ohh, yeah that makes a lot more sense, i changed that.

**Comment by onedr0p** [comment]
Looks good here @ojsef39 Thanks!

I'll probably end up refactoring the workflow at some point to support creating release images.

**Review by onedr0p — APPROVED** [review]

**Comment by ojsef39** [comment]
> I'll probably end up refactoring the workflow at some point to support creating release images.

Yeah ig that would make sense because GitHub keeps all the builds, i thought it would overwrite.

**Pull request merged**